src/pages/support/tracing.md

```diff
@@ -1,21 +1,39 @@
 # Tracing Adobe I/O Events
```

Comment:
Can you also update the snapshot at https://developer.adobe.com/events/docs/guides/#asynchronous-validation?

Reply:
Good catch. Updated.